jest.config.js

```diff
@@ -0,0 +1,5 @@
+export default {
+  maxWorkers: 1,
+  testPathIgnorePatterns: ["/node_modules/", "\\.ts$", "\\.tsx$"],
```

Comment:
This seems to be excluding the `*.test.ts` files also, which may not be what's intended? But even with this filter removed, trying to actually run the `index.test.ts` file doesn't succeed, "unexpected token" shenanigans that aren't trivially resolvable.

Reply:
Yes, I had more success building the app (just run `tsc`) and then having jest run the compiled test `dist/index.test.js`.

If @arthyn or someone else has any advice on how to run the ts files themselves I'm happy to fix it.